RevenueCatUI/Templates/Template4View.swift

```diff
@@ -214,9 +257,13 @@ struct Template4View: TemplateViewType {
                 return 1.5
             }
         }()
+        let maximumPackagesToDisplay = 3
```

Comment:
We weren't enforcing a limit, which would have shrunk packages unnecessarily when displaying more than 3, instead of making it scrollable.